staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -61,6 +61,16 @@ const (
 	// Allows us to enable anonymous auth for only certain apiserver endpoints.
 	AnonymousAuthConfigurableEndpoints featuregate.Feature = "AnonymousAuthConfigurableEndpoints"
 
+	// owner: @stlaz @tkashem @dgrisonnet
+	// alpha: v1.32
```

Comment:
drop the alpha release comment, it's included in the versioned registration below

Reply:
yes, dropped